manifests/type/containerization/kubernetes/kubeconfig.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         "properties": {
           "config": {
             "$id": "#/properties/config",
-            "type": "string",
+            "type": "object",
             "title": "contains Kubernetes Kubeconfig"
```

Comment:
Is it always should be the `object` and it is some old bug or do we changed something?

Reply:
yep, it's an old bug